pkg/proxy/ipvs/proxier_test.go

```diff
@@ -430,12 +426,12 @@ func TestGetNodeIPs(t *testing.T) {
 		},
 		// case 8
 		{
-			devAddresses: map[string][]string{"kube-ipvs0": {"1.2.3.4", "2.3.4.5"}, "eth5": {"3.4.5.6"}, "lo": {"127.0.0.1"}},
+			devAddresses: map[string][]string{"kube-ipvs0": {"1.2.3.4", "2.3.4.5"}, "eth5": {"3.4.5.6"}},
 			expectIPs:    []string{"3.4.5.6"},
 		},
 		// case 9
 		{
-			devAddresses: map[string][]string{"ipvs0": {"1.2.3.4"}, "lo": {"127.0.0.1"}, "encap7": {"10.20.30.31"}},
```

Comment:
why removing the loopbacks here, shouldn't those be excluded by the new methods `GetAllLocalAddresses()`?

Reply:
Because the GetAllLocalAddresses() comes from the "fake" in testing/fake.go, and that does not filter loopback (code was moved). I could add filtering in "fake", but then I would like test the test-code.

Reply:
But you are right. It is clearer to have it in the test-case if the mock procedure would change in the future. I'll restore it and add some ipv6 tests for good measure.

But I will not be able to do that today.